columnq/src/table/mod.rs

```diff
@@ -286,13 +288,19 @@ impl TableSource {
             io_source: source.into(),
             schema: None,
             option: None,
+            batch_size: 1024,
```

Comment:
Better keep the magic number in a single place :)

```suggestion
            batch_size: Self.default_batch_size(),
```

Reply:
Good catch, thanks